.travis.yml

```diff
@@ -3,7 +3,8 @@ os:
   - linux
   - osx
 node_js:
-  - "6"
+  - "8"
+  - "10"
```

Comment:
@imrehg, the thing is that v10.x.x of the CLI is still expected to work with Node 6. We have a v11-meta branch on which we are updating the travis and appveyor yml files, check this PR:
https://github.com/balena-io/balena-cli/pull/1197/files

v11 of the CLI will target Node 8+ and leave Node 6 behind. We've had to drop the balenaCI test for Node 6 because balenaCI dropped support for Node 6, but the CLI v10 didn't drop support for Node.